### ISSUE TITLE
Mergerfs.balance - python issue

### ISSUE THREAD

**Issue by Symbiot78** [trapexit/mergerfs-tools#58]
Am suddenly seeing alot of this error when I try to run Mergerfs.balance

` File "/usr/local/bin/mergerfs.balance", line 64, in mergerfs_control_file
    return mergerfs_control_file(dirname)
  File "/usr/local/bin/mergerfs.balance", line 64, in mergerfs_control_file
    return mergerfs_control_file(dirname)
  File "/usr/local/bin/mergerfs.balance", line 64, in mergerfs_control_file
    return mergerfs_control_file(dirname)
  File "/usr/local/bin/mergerfs.balance", line 59, in mergerfs_control_file
    ctrlfile = os.path.join(basedir,'.mergerfs')
  File "/usr/lib/python3.4/posixpath.py", line 75, in join
    sep = _get_sep(a)
  File "/usr/lib/python3.4/posixpath.py", line 39, in _get_sep
    if isinstance(path, bytes):
RuntimeError: maximum recursion depth exceeded while calling a Python object
`

Command I am running:

` mergerfs.balance -e aquota.user -e aquota.group /media/6a874986-7a03-4fe8-b9fb-3fbfc6cc3e95/`

**Comment by trapexit** [comment]
Is /media/6a874986-7a03-4fe8-b9fb-3fbfc6cc3e95/ the mergerfs mount?

It should handle the situation better but it doesn't appear to me you've pointed it to mergerfs mount or somewhere within it.

**Comment by Symbiot78** [comment]
I am a complete klutz. 
I noticed an off balance in space used and checked my scheduled task where I found my merger balance command.
I just forgot that I moved it to crontab and not the GUI scheduled tasks.

When I ran:

`mergerfs.balance -s 10M -e "*aquota*" -e "snapraid*" -E "*docker/*" -E "*downloads/*" /srv/b4e5810f-00ae-4677-beff-3e79aedfafd8/`

Everything worked.

The other one was just something I moved when I moved my OMV instance to new hardware.

Sorry for the trouble.
